python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -90,12 +90,7 @@ def __init__(self, entity_model, search_label, allow_task_creation, parent):
         self._ui.entity_tree.expanded.connect(self._on_item_expanded)
         self._ui.entity_tree.collapsed.connect(self._on_item_collapsed)
 
-        # create the overlay 'busy' widget that will be displayed when the model is reset:
-        self._overlay_widget = ShotgunModelOverlayWidget(None, self._ui.entity_tree)
```

Comment:
Removed because it has never been working with the app and I didn't want to have anything that we didn't fully understand mess around with models since they are so brittle/leak friendly in certain DCCs.